src/js/letters/actions/letters.js

```diff
@@ -12,9 +12,25 @@ import {
   GET_LETTER_PDF_FAILURE,
   GET_LETTER_PDF_SUCCESS,
   LETTER_ELIGIBILITY_ERROR,
-  UPDATE_BENFIT_SUMMARY_REQUEST_OPTION
+  UPDATE_BENFIT_SUMMARY_REQUEST_OPTION,
+  UPDATE_ADDRESS_SUCCESS,
+  UPDATE_ADDRESS_FAILURE
 } from '../utils/constants';
 
+// Copied from the vets-api v0/address POST swagger; TODO: delete once
+// actual form data is passed to updateAddress
+const exampleAddress = {
+  type: 'DOMESTIC',
```

Comment:
Are the only options for `type` `DOMESTIC` and `MILITARY`? Wondering how best we're going to send this field because we're not actually including a question for it. I'm guessing it will be dependent on if they select the military address options in the state dropdown, and then we can automatically update this field based on that. Might need a separate action to do that though? I think this will be easier to do once adding the fields is done, so we can include that later.

Reply:
there's an `INTERNATIONAL` type too

Reply:
Now tracked in https://github.com/department-of-veterans-affairs/vets.gov-team/issues/4520